glide.lock

```diff
@@ -286,6 +279,10 @@ testImports:
   version: d8ed2627bdf02c080bf22230dbb337003b7aba2d
   subpackages:
   - difflib
+- name: github.com/Sirupsen/logrus
```

Comment:
we keep getting this issue because `Sirupsen` got renamed to `sirupsen`. Can we make sure what we use the lower-case package name everywhere?

Reply:
Goodness. Yeah, just fixed it. The most annoying change in go's infrastructure if you ask me

Reply:
Hmm. Well, it works locally now, but failing on travis